src/main/java/org/jabref/gui/entryeditor/EntryEditorTab.java

```diff
@@ -39,7 +38,7 @@ public void notifyAboutFocus(BibEntry entry) {
         if (!entry.equals(currentEntry) || !currentEntryType.equals(entry.getType())) {
             currentEntry = entry;
             currentEntryType = entry.getType();
-            DefaultTaskExecutor.runInJavaFXThread(() -> bindToEntry(entry));
```

Comment:
I am not sure if you can already do this. As far as I remember the import inspection or other swing dialogs will raise an fx thread error

Reply:
This is necessary so that the entry editor is always bound to correct entry before other events hit it. If there are problems with certain dialogs, then those need to be fixed (call runInJavaFXThread there). I've not yet noticed any issues.